docs/remote/attach-container.md

```diff
@@ -0,0 +1,79 @@
+g---
+Order: 9
+Area: remote
+TOCTitle: Attach to a Running Container
+PageTitle: Attach to a running container using Visual Studio Code Remote Development
+ContentId: ed14ef07-f44c-4675-b95b-cb5faffc7abb
+MetaDescription: Attach to a running container using Visual Studio Code Remote Development
+DateApproved: 7/9/2020
+---
+# Attach to a running container
+
+While using Visual Studio Code to spin up a new container can be useful in many situations, it may not match your workflow and you may prefer to "attach" VS Code to an already running Docker container - regardless of how it was started. Once attached, you can install extensions, edit, and debug like you can when you open a folder in a container using [devcontainer.json](/docs/remote/devcontainerjson-reference.md).
```

Comment:
"spin up a new container" could be a reference to the other containers article since if you jump right here you may not know what that means